kolibri/core/auth/models.py

```diff
@@ -923,6 +925,25 @@ def is_member_of(self, coll):
             return self.facility_id == coll.id
         return Membership.objects.filter(user=self, collection=coll).exists()
 
+    def delete_imported_user(self):
+        from kolibri.core.auth.utils.delete import DisablePostDeleteSignal
+
+        with DisablePostDeleteSignal(), transaction.atomic():
```

Comment:
Looking at the deletion utilities: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/auth/utils/delete.py

There's other cleanup we should be doing here. I think maybe it would be better to construct a user focused deletion function in that utility module, rather than adding it as a model method. We should also follow the more complete deletion that is done in the delete.py module, deleting the related models etc.

Reply:
Done!